src/script/interpreter.cpp

```diff
@@ -16,8 +16,6 @@
 
 using namespace std;
 
-typedef vector<unsigned char> valtype;
```

Comment:
I don't like exposing `valtype` externally, the type name is too general.  Would prefer to change the function signatur to just
```c++
bool CheckSignatureEncoding(const vector<unsigned char> &vchSig, unsigned int flags, ScriptError* serror) {
```